src/IPinfo.php

```diff
@@ -53,7 +55,9 @@ public function __construct($access_token = null, $settings = [])
         $this->http_client = new Client($guzzle_opts);
 
         $countries_file = $settings['countries_file'] ?? self::COUNTRIES_FILE_DEFAULT;
+        $eu_countries_file = $settings['eu_countries_file'] ?? self::EU_COUNTRIES_FILE_DEFAULT;
         $this->countries = $this->readCountryNames($countries_file);
+        $this->eu_countries = $this->readCountryNames($eu_countries_file);
```

Comment:
Need a more generic name for this function if we're gonna reuse it for this. I realized we didn't do this in the Python SDK case, but that's alright, I just fixed it there: https://github.com/ipinfo/python/commit/5253fd09eb04c317efb954bd658c6ab24ea11dc8